app/repositories/contactdetails/ContactDetailsRepository.scala

```diff
@@ -60,4 +62,15 @@ class ContactDetailsMongoRepository(implicit mongo: () => DB)
       case None => throw ContactDetailsNotFound(userId)
     }
   }
+
+  def findUserIdByEmail(email: String): Future[String] = {
+    val query = BSONDocument("contact-details.email" -> email)
```

Comment:
Do we need to ensure case sensitivity here?  That may be taken care of by the FE caller.